Plugins~/Src/MeshSyncClientBlender/msblenBinder.cpp

```diff
@@ -375,7 +375,12 @@ bool blender::BObject::is_selected() const
 
 Mesh* BObject::to_mesh() const
 {
+#if BLENDER_VERSION >= 303
+    // In blender 3.3 the pointer to interpret the bool moves by 8 instead of 1 for some reason so this needs to be a type of that size:
```

Comment:
Is this change specific to to_mesh or is it affecting other parts?

Reply:
I only found the issue in this call.